contracts/src/scripts/DeployLiquity2.s.sol

```diff
@@ -893,6 +907,7 @@ contract DeployLiquity2Script is DeployGovernance, UniPriceConverter, StdCheats,
                 string.concat('"boldToken":"', address(deployed.boldToken).toHexString(), '",'),
                 string.concat('"hintHelpers":"', address(deployed.hintHelpers).toHexString(), '",'),
                 string.concat('"multiTroveGetter":"', address(deployed.multiTroveGetter).toHexString(), '",'),
+                string.concat('"exchangeHelpers":"', address(deployed.exchangeHelpers).toHexString(), '",'),
```

Comment:
Should we add the address of the newly deployed `exchangeHelpers` to the existing manifest at [contracts/addresses/11155111.json](https://github.com/liquity/bold/blob/main/contracts/addresses/11155111.json)?